Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs

```diff
@@ -161,6 +161,8 @@ sealed class InterpreterListToolWindow : ToolWindowPane {
                     view.Factory.Configuration,
                     false
                 ));
+            } catch (DirectoryNotFoundException) {
```

Comment:
Do we care if directory is there but is empty?

Reply:
That's fine. The way this is used is we enumerate all *.py files in that folder, and execute them in alphabetical order when the REPL starts. If there's no files in the folder, we have nothing to execute.